thundra/integrations/botocore.py

```diff
@@ -50,7 +50,7 @@ def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
 
         scope.span.domain_name = Constants.DomainNames['DB']
         scope.span.class_name = Constants.ClassNames['DYNAMODB']
-        scope.span.operation_name = 'dynamodb: ' + str(self.request_data['TableName'])
+        scope.span.operation_name = str(self.request_data['TableName'])
```

Comment:
what if there is no `TableName` in the request? For example, when the request is for list tables